packages/create-bison-app/template/tailwind.config.js

```diff
@@ -0,0 +1,33 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
I saw Tailwind now supports TypeScript config files: https://tailwindcss.com/blog/tailwindcss-v3-3#esm-and-typescript-support

Reply:
I'm gonna stick with .js, since with the JSDoc comment it provides the same benefits as .ts, without the potential for tooling headaches.